src/modules/keyboardmanager/common/ErrorTypes.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-namespace KeyboardManagerHelper
+namespace Helpers
```

Comment:
Let's not use a namespace here, instead use a descriptive name for the enum: `ErrorType` -> `ShortcutErrorType`